assets/tests/components/__snapshots__/propertiesPanel.test.tsx.snap

```diff
@@ -154,28 +154,150 @@ Array [
+              className="m-tabs__tab"
+              id="react-tabs-8"
+              role="tab"
+              tabIndex={null}
+            >
+              <svg
+                className="m-tabs__tab-status-icon"
+                height="14"
+                width="14"
+              >
+                <circle
+                  className="m-tabs__tab-status-icon-outer-circle"
+                  cx="7"
+                  cy="7"
+                  r="6"
+                />
+                <circle
+                  className="m-tabs__tab-status-icon-inner-circle"
+                  cx="7"
+                  cy="7"
+                  r="4"
+                />
+              </svg>
+              Run
+            </li>
+            <li
+              aria-controls="react-tabs-11"
+              aria-disabled="false"
+              aria-selected="false"
+              className="m-tabs__tab"
+              id="react-tabs-10"
+              role="tab"
+              tabIndex={null}
+            >
+              <svg
+                className="m-tabs__tab-status-icon"
+                height="14"
+                width="14"
+              >
+                <circle
+                  className="m-tabs__tab-status-icon-outer-circle"
+                  cx="7"
+                  cy="7"
+                  r="6"
+                />
+                <circle
+                  className="m-tabs__tab-status-icon-inner-circle"
+                  cx="7"
+                  cy="7"
+                  r="4"
+                />
+              </svg>
+              Block
+            </li>
+          </ul>
+          <div
+            aria-labelledby="react-tabs-6"
+            className="m-tabs__tab-panel m-tabs__tab-panel--selected"
+            id="react-tabs-7"
+            role="tabpanel"
+          >
+            <div
+              className="m-properties-list"
+            >
+              <table
+                className="m-properties-list__table"
               >
-                123-0123
-              </td>
-            </tr>
-          </tbody>
-        </table>
+                <tbody>
+                  <tr
+                    className="m-properties-list__property "
+                  >
+                    <td
+                      className="m-properties-list__property-label"
+                    >
+                      Run
+                    </td>
+                    <td
+                      className="m-properties-list__property-value"
+                    >
+                      123-0123
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+          <div
+            aria-labelledby="react-tabs-8"
+            className="m-tabs__tab-panel"
+            id="react-tabs-9"
+            role="tabpanel"
+          />
+          <div
+            aria-labelledby="react-tabs-10"
+            className="m-tabs__tab-panel"
+            id="react-tabs-11"
```

Comment:
I'm a little worried about global state making these snapshots fragile. When doing a single `StatusRunBlockTabs`, the 6 components get ids 0-5. But this file has 2 tests, one with ids 0-5 and one with 6-11. Is there risk of these ids changing from a distance and causing intermittent test failures?

Reply:
I've run the tests many times and haven't seen a problem at this point.